lib/components/narrative/default/access-leg.js

```diff
@@ -1,12 +1,10 @@
-import React, {Component} from 'react'
+import coreUtils from '@opentripplanner/core-utils'
 import PropTypes from 'prop-types'
+import React, {Component} from 'react'
 
 import Icon from '../icon'
-
 import LegDiagramPreview from '../leg-diagram-preview'
 import { distanceString } from '../../../util/distance'
```

Comment:
Let's use `@opentripplanner/humanizeDistance`. I think it's ok if it's in this same PR (unless you have it in some other PR already). Not 100% sure why that's a separate package... I think that just leads to confusion/oversights like this ^^.